src/Form/FormRadio.jsx

```diff
@@ -0,0 +1,74 @@
+/* disable label has for since we are leveraging implicit labeling */
+/* eslint-disable jsx-a11y/label-has-for */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { resolveTextType } from './FormText';
+import { useRadioGroupContext } from './FormRadioGroupContext';
+import FormRadioFeedback from './FormRadioFeedback';
+
+const FormRadio = React.forwardRef(({
+  children,
+  className,
+  controlClassName,
+  labelClassName,
+  description,
+  isInvalid,
+  isValid,
+  ...props
+}, ref) => {
+  const { getRadioInputProps } = useRadioGroupContext();
+  const radioInputProps = getRadioInputProps(props);
+
+  return (
+    <label
+      className={classNames('pgn__form-radio', className, {
+        'pgn__form-radio-valid': isValid,
+        'pgn__form-radio-invalid': isInvalid,
+        'pgn__form-radio-disabled': radioInputProps.disabled,
+      })}
+    >
+      <span className={classNames('pgn__form-radio-lockup', controlClassName)}>
+        <input
+          {...props}
+          {...radioInputProps}
+          type="radio"
+          ref={ref}
+        />
+        <span className={classNames('pgn__form-radio-control', controlClassName)} />
```

Comment:
Shouldn't the `pgn__form-radio-control` class name be on the actual input?  What's it mean that it's on this empty `span` between the input and the label?

Reply:
We are unable to style the input itself so this element is the visual replacement of the control. Could name it `pgn__form-radio-faux-control` if we felt it was important for clarity